index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-main</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
```

Comment:
Page is overflowing horizontally. Please make sure that no content is overflowing from its container.